packages/react/src/components/Geo/MapView/__tests__/MapView.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Amplify, Auth } from 'aws-amplify';
+
+import { MapView } from '../index';
+
+import { act } from 'react-dom/test-utils';
+
+jest.mock('maplibre-gl-js-amplify', () => {
+  return {
+    AmplifyMapLibreRequest: jest.fn().mockImplementation(() => {
+      return {
+        amplifyTransformRequest: true,
+        transformRequest: (url, resourceType) => {
+          if (resourceType === 'Style' && !url.includes('://')) {
+            url = `https://maps.geo.us-east-2.amazonaws.com/maps/v0/maps/${url}/style-descriptor`;
+          }
+          if (new URL(url).hostname.endsWith('.amazonaws.com')) {
+            return {
+              url: 'https://maps.geo.us-east-2.amazonaws.com/maps/v0/maps/map5df169f7-staging/style-descriptor',
+            };
+          }
+        },
+      };
+    }),
+  };
+});
+
+jest.mock('react-map-gl', () => () => {
```

Comment:
Can we add a comment why we need to mock this for future readers?

Reply:
added comment in latest commit